docs/source/tutorials/multiannotator.ipynb

```diff
@@ -0,0 +1,555 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "98042e7f",
+   "metadata": {},
+   "source": [
+    "Here, we use the `multiannotator.get_label_quality_multiannotator()` function which returns a dictionary containing three items:\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "76d7c0e2",
+   "metadata": {},
+   "source": [
+    "- `label_quality_multiannotator` which gives us the improved consensus labels using information from each of the annotators and the model. The DataFrame also contains information about the number of annotations, annotator agreement and consensus quality score for each example.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "bf945113",
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"label_quality_multiannotator\"].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "984d65c4",
+   "metadata": {},
+   "source": [
+    "- `detailed_label_quality` which returns the label quality score for each label given by every annotator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "14251ee0",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"detailed_label_quality\"].head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "db02e63d",
+   "metadata": {},
+   "source": [
+    "- `annotator_stats` which gives us the annotator quality score for each annotator, alongisde other information such as the number of examples each annotator labeled, their agreement with the consensus label and the class they perform the worst at. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "efe16638",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "multiannotator_dict[\"annotator_stats\"].head(10)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a0d09bfa",
+   "metadata": {},
+   "source": [
+    "The `annotator_stats` DataFrame is sorted by increasing `annotator_quality`, showing us the worst annotators first.\n",
+    "\n",
+    "Notice that in the above table annotators with ids 45 to 49 have the worst annotator quality score, which is expected because we made the last 5 annotators systematically worse than the rest."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "20ca8dd2",
+   "metadata": {},
+   "source": [
+    "### Comparing improved consensus labels"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1b49657d",
+   "metadata": {},
+   "source": [
+    "We can get the improved consensus labels from the `label_quality_multiannotator` DataFrame shown above."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "abd0fb0b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "improved_consensus_label = multiannotator_dict[\"label_quality_multiannotator\"][\"consensus_label\"]"
```

Comment:
Type of `improved_consensus_label` does not appear to match `majority_vote_label`?

I think we should make them have the same type by default, probably np.ndarray